### ISSUE TITLE
[BUG] anim_translate and actions don't sync up well?

### ISSUE THREAD

**Issue by Sov-trotter** [Wikunia/Javis.jl#369]
For the code below : 
```julia
using Javis

function ground(args...)
    background("black")
    sethue("white")
end

astar(args...; do_action=:stroke) = star(O, 50, 5, 0.5, 0, do_action)
acirc(args...; color = "red", do_action=:stroke) = circle(Point(0, 0), 50, do_action)

video = Video(500, 500)
back = Background(1:50, ground)
star_obj = Object(1:50, astar)
act!(star_obj, Action(10:25, morph_to(acirc)))
act!(star_obj, Action(26:35, anim_translate(Point(100, -100))))
act!(star_obj, Action(36:45, morph_to(astar)))


render(video, pathname = "testmorph.gif")
```
Expected Behaviour: 
So what we want is the star to morph into a circle, then that circle should translate to (100, -100)(the Javis CRS)
and the morph back to a star positioned at O(0, 0).

Current Behaviour: 
So translate() shifts the origin to 100, -100 and is never shifted back to 0,0.
![testmorph](https://user-images.githubusercontent.com/43717431/126467427-12da380e-1acb-4849-87f4-efb387f9bd6e.gif)

**Comment by Wikunia** [comment]
Thanks for opening the issue. I feel like this might be more a problem of morphing as no new object is created during the morph I feel like all actions after morphing might potentially be broken :disappointed:

**Comment by Wikunia** [comment]
The docstring is simply wrong and fixed in: https://github.com/Wikunia/Javis.jl/pull/370

**Comment by Sov-trotter** [comment]
So this is expected behaviour?

**Comment by Wikunia** [comment]
Yes it will move by a vector instead of to a point. The other one is currently not really possible as one does not know where the point is at the beginning and it would have some unsolvable problems when one applies two of those actions at the same time. See also #367

**Comment by Sov-trotter** [comment]
AH got it thanks!
